src/meta/expressionEvaluator/functions/validatorEqualToSum.ts

```diff
@@ -10,14 +10,29 @@ export const validatorEqualToSum: ExpressionFunction<Context> = {
   name: 'validatorEqualToSum',
   minArity: 2,
   executor: () => {
-    return (value: string, otherValues: Array<string>): NodeValueValidation => {
+    return (
+      value: string,
+      otherValues: Array<string>,
```

Comment:
What is otherValues ?

Reply:
otherValues is the array of values to compare (usually, the subcategories). I can rename it to childValues if it is clearer

Reply:
Aaa ok , let me think a bit about this . No worries for now . 
I'll resolve this comment .